android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1517,6 +1560,27 @@ public void setAllowsProtectedMedia(boolean enabled) {
     }
   }
 
+  /**
+   * A {@link FrameLayout} container to hold the {@link RNCWebView}.
+   * We need this to prevent WebView crash when the WebView is out of viewport and
+   * {@link com.facebook.react.views.view.ReactViewGroup} clips the canvas.
+   * The WebView will then create an empty offscreen surface and NPE.
+   */
+  public static class RNCWebViewWrapper extends FrameLayout {
+    RNCWebViewWrapper(Context context, RNCWebView webView) {
+      super(context);
+      // We make the WebView as transparent on top of the container,
+      // and let React Native sets background color for the container.
+      webView.setBackgroundColor(Color.TRANSPARENT);
```

Comment:
the default WebView's background color is [white](https://chromium.googlesource.com/chromium/src/+/main/android_webview/java/src/org/chromium/android_webview/AwContents.java#410), we should set it as transparent so that the [background example](https://github.com/react-native-webview/react-native-webview/blob/master/example/examples/Background.tsx) will work.

Reply:
@FatemaDholkawala you can use https://www.npmjs.com/package/patch-package

You can find more instructions in their readme or by googling how to use it.

Reply:
@andreialecu  Thank you so much

Reply:
Ah, so the `backgroundColor` `style` attribute is being applied to the `FrameLayout` instead of the `WebView` with this change?

Reply:
That was my assumption too.

Reply:
@andreialecu  Thanks

Reply:
> Ah, so the backgroundColor style attribute is being applied to the FrameLayout instead of the WebView with this change?

yes exactly, as @andreialecu said.